lib/Parse/ParseDecl.cpp

```diff
@@ -5092,6 +5089,9 @@ ParserResult<ClassDecl> Parser::parseDeclClass(SourceLoc ClassLoc,
     Scope S(this, ScopeKind::ClassBody);
     ParseDeclOptions Options(PD_HasContainerType | PD_AllowDestructor |
                              PD_InClass);
+    if (Flags & PD_InIfConfig) {
+      Options |= PD_InIfConfig;
```

Comment:
There was no flag for this before.  It used to be a function of "scope", but I felt it was inappropriate for scope to ask the question "is this a condition" let alone "is this an inactive condition" like it did before.